deepspeed/runtime/bf16_optimizer.py

```diff
@@ -94,6 +94,15 @@ def __init__(self,
 
         see_memory_usage('end bf16_optimizer', force=True)
 
+    def destroy(self):
+        for i, _ in enumerate(self.optimizer.param_groups):
+            for p in self.bit16_groups[i]:
```

Comment:
FYI @chiragjn - could you run the pre-commit formatter, that will resolve the formatting tests.  Thanks!

Reply:
Done :)